### ISSUE TITLE
A video stopped playing properly

### ISSUE THREAD

**Issue by psprint** [asciinema/asciinema-player#57]
Hello,
this video was playing properly for months:

https://asciinema.org/a/48490

Now, starting from 12 second, artifacts are there on screen.

**Comment by sickill** [comment]
Hmm, this is interesting. Can you tell when it became like this, or you just noticed now?

**Comment by psprint** [comment]
I've noticed now. It was for sure working earlier, I've watched the video multiple times after uploading and also later.

**Comment by michael-lazar** [comment]
One of my videos is having a similar issue. I'm sure it worked when I uploaded it ~9 months ago, but now I'm seeing a bunch of artifacts. I tried downloading the asciicast and it was perfect when I played it locally.

https://asciinema.org/a/81251

- at 0:06 the ``2.`` is shifted to the left by one character
- at 0:10 the scrolling cursor is totally messed up
- at 0:53 the ``2.`` is once again shifted to the left
- starting at 1:33 there are a couple of pages with lines shifted to the left

It looks it happens whenever there's an indented line after a blank line?
